app/src/main/java/io/github/omisie11/spacexfollower/ui/launches/LaunchesFragment.kt

```diff
@@ -31,7 +35,19 @@ class LaunchesFragment : Fragment(R.layout.fragment_recycler_sorting),
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-
+        enterTransition = MaterialContainerTransform().apply {
+            startView = requireActivity().findViewById(R.id.text_label_launches)
+            endView = root
+            duration = resources.getInteger(R.integer.core_motion_duration_large).toLong()
+            scrimColor = Color.TRANSPARENT
+            containerColor = requireContext().themeColor(R.attr.colorSurface)
+            startContainerColor = requireContext().themeColor(R.attr.colorSurface)
+            endContainerColor = requireContext().themeColor(R.attr.colorSurface)
+        }
```

Comment:
I think it would be nice nice to extract it into private function to keep onViewCreated more readable.

Reply:
okay I will extract it

Reply:
Thanks, but it is not crucial :)